registration/registration-services/src/main/java/io/mosip/registration/mdm/spec_0_9_5/service/impl/MosipDeviceSpecification_095_ProviderImpl.java

```diff
@@ -241,8 +241,8 @@ public InputStream stream(MdmBioDevice bioDevice, String modality) throws RegBas
 
 					mosipDeviceSpecificationHelper.validateQualityScore(dataDTO.getQualityScore());
 					
-					if (dataDTO.getTransactionId() == null
-							|| !dataDTO.getTransactionId().equalsIgnoreCase(rCaptureRequestDTO.getTransactionId())) {
+					if (dataDTO.getTransactionId() == null && rCaptureRequestDTO !=null &&
```

Comment:
in either of the condition, capture should fail, so instead of && it should be || between the conditions.